nix/module.nix

```diff
@@ -182,7 +178,7 @@ in {
       after = ["network.target" "postgresql.service"];
       requires = ["postgresql.service"];
       restartIfChanged = true;
-      path = with pkgs; [postgresql_11 liberaforms-env openssl];
+      path = with pkgs; [postgresql_11 penv openssl];
```

Comment:
nixpkgs already has PostgreSQL service defined, we just need to configure it. no need to define a new one